### ISSUE TITLE
error during flutter run

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/conference_app#33]
I see this error when doing a flutter run for android:

```
Exit code 1 from: /Users/devoncarew/projects/dart-lang/conference_app/android/gradlew app:properties:
           
           FAILURE: Build failed with an exception.
           
           * Where:
           Build file '/Users/devoncarew/projects/dart-lang/conference_app/android/app/build.gradle' line: 20
```

That's this line: `properties.load(new FileInputStream(propertiesFile))`.

@rock3r, perhaps we can make the gradle script more tollerant of a missing signing/debug.properties file?

**Comment by rock3r** [comment]
I'll take a stab at this tomorrow, but I suspect that it is actually about the release keystore missing. The debug files are both in VCS. Easiest workaround for now is to duplicate the `debug` keystore and properties file and rename them to `release`

**Comment by rock3r** [comment]
This should be addressed by #34 :)
